CyrillicToTranslit.js

```diff
@@ -74,15 +74,17 @@ module.exports = function cyrillicToTranslit(config) {
       return "";
     }
 
+    const normalizedInput = input.normalize()
```

Comment:
please add missing `;`

Reply:
Also would be great to have a comment with some clarification before this line of code why we normalize the string and what we want to achieve.